const { failed } = require("../utils/errorHandler")


exports.updateUserDetails = async(req, res)=>{
    try{

    }catch(err){
        failed(res, err);
    }
}
exports.updateProfilePicture = async(req, res)=>{
    try{

    }catch(err){
        failed(res, err);
    }
}
exports.getUserProfile = async(req, res)=>{
    try{

    }catch(err){
        failed(res, err);
    }
}
exports.getBookedHotels = async(req, res)=>{
    try{

    }catch(err){
        failed(res, err);
    }
}
exports.getNotifications = async(req, res)=>{
    try{

    }catch(err){
        failed(res, err);
    }
}